packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -744,6 +766,7 @@ export class AtomicSearchBox {
             ref={(el) => (this.inputRef = el as HTMLInputElement)}
             bindings={this.bindings}
             value={this.searchBoxState.value}
+            title={this.tooltipText}
```

Comment:
wonder if `title` and `ariaLabel` could be combined in some way because[ looks like aria label is prioritized over title by screen readers](https://stackoverflow.com/a/27953625) so the title message wouldn't be accessible

Reply:
refactored to reuse the same msg for both title and arialabel